uber/config.py

```diff
@@ -352,6 +352,14 @@ def PAGE_PATH(self):
     def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
+    @property
+    def IS_DEALER_REG_REQUEST(self):
+        is_dealer_referer = os.path.basename(cherrypy.request.headers.get('Referer', '')) in ('dealer_registration', 'post_form')
+        is_dealer_post = c.HTTP_METHOD == 'POST' and c.PAGE_PATH == '/preregistration/post_form' and is_dealer_referer
```

Comment:
Yeah, so you're right about inspecting the parameters for `PSEUDO_DEALER_BADGE`. If we do that then we can take the other check out of `preregistration.form()`. This was a quick fix, so as long as we correct it before prereg, I'm okay with it. I've created an issue to make the changes you suggested (see https://github.com/magfest/ubersystem/issues/2758).

I'll be able to get to it tomorrow. As for tonight? Game of Thrones, baby!